admin/view/wp-slimstat-reports.php

```
@@ -1062,6 +1066,33 @@ public static function report_pagination($_count_page_results = 0, $_count_all_r
+     * count($all_results) shrinks on later pages. We use count_records()
+     * for the true total, but cap it to limit_results because get_recent()
+     * can only access that many rows via SQL. The existing "200+" suffix
+     * in report_pagination() signals when more rows exist beyond the cap.
+     *
+     * For type=top reports, SQL fetches the full set (no OFFSET) and
+     * count($all_results) is already stable.
+     */
+    private static function get_report_total_count($_args, $all_results)
+    {
+        if (!empty($_args['type']) && $_args['type'] === 'recent') {
+            $true_total = wp_slimstat_db::count_records(
```

PHPStan (2.1.46)

[error] Call to an undefined static method wp_slimstat_db::count_records().


(staticMethod.notFound)